client/src/views/Settings/Tabs/EditProfile/index.vue

```diff
@@ -13,13 +13,15 @@
         <q-card-section>
           <template v-if="!isNewAvatarSelected">
             <div class="row justify-center">
-              <avatar :img="photoUrl" />
+              <avatar :img="profile.avatar_url" />
             </div>
           </template>
           <template v-else>
             <edit-image
               :src="photoUrl"
               class="avatar"
+              ref="avatarEditor"
+              @on-created-blob="onCreatedAvatarBlob"
```

Comment:
Naming is ok for now, but after demo please rename it (always omit useless things in your namings, event or callback is always verb in the past aka "[item][action]d" or "blobCreated"; things like "on", "event" don't really help much when used in names)